external/netfx/netfx.depproj

```diff
@@ -21,27 +21,9 @@
       <RefPath Condition="'$(ForShims)' != 'true'">$(RefPath)</RefPath>
       <RefPath Condition="'$(ForShims)' == 'true'">$(NetFxRefPath)</RefPath>
     </BinplaceConfiguration>
-  </ItemGroup>
-
-  <!-- Filter the targeting pack to just these assemblies which we need netcoreapp shims for -->
-  <ItemGroup>
-    <TargetingPackReference Include="mscorlib" />
```

Comment:
I'm interested in understanding why the filtered approach didn't work still. My expectation is that our builds didn't need the full targeting pack and would likely be able to get away with just a smaller subset.